test/python/gemm/frost/test_block_scale_matmul.py

```
@@ -1850,13 +1850,15 @@ def _sm107_kw(config_name, cta_group=1):
 
 def test_catalog_has_sm107_geometries():
     sm107 = [c for c in CATALOG if c.pipeline == "sm107"]
-    # 2 cta_n × the shared 15-cluster enumeration.
-    assert len(sm107) == 30
-    pat = re.compile(r"^CONFIG_sm107_128x(128|256)x128_128x(128|256)x64_cluster\d+x\d+$")
+    # num_mma_m {1,2} × 2 cta_n × the shared 15-cluster enumeration.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)